.github/workflows/main.yml

```diff
@@ -8,9 +8,9 @@ concurrency: Testing
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
```

Comment:
This comment says 

> only for the master branch

but here two other.

Reply:
I investigated this question, since each branch has its own workflow, it makes no sense to additionally specify other workflows for this branch. I have already made these changes